src/sentry/static/sentry/app/components/narrowLayout.jsx

```diff
@@ -29,7 +30,7 @@ class NarrowLayout extends React.Component {
     return (
       <div className="app">
         <div className="pattern-bg" />
-        <div className="container">
+        <div className="container" style={{maxWidth: this.props.maxWidth}}>
```

Comment:
setting max-width in a styled component didn't seem to be overriding the value in `layout.less`

Reply:
Yeah there can be some weird precedence things